blockchain/chainio.go

```diff
@@ -30,6 +30,10 @@ const (
 	// currentDatabaseVersion indicates what the current database
 	// version is.
 	currentDatabaseVersion = 1
+
+	// minimumStxoSize is the absolute smallest size a serialized STXO
+	// may be, below which is assumed to be an error.
+	minimumStxoSize = 4
```

Comment:
I assume minimumStxoSize is due to 1 coinbase + 3 stakebases, correct?  Maybe add that in the comment or why it is 4.